docs/source/deploypeer/peerplan.md

```diff
@@ -0,0 +1,94 @@
+# Planning for a production peer
+
+Audience: Architects, network operators, users setting up a production Fabric network who are familiar with Transport Layer Security (TLS), Public Key Infrastructure (PKI) and Membership Service Providers (MSPs).
+
+Peer nodes are a fundamental element of a Fabric network because they host ledgers and smart contracts that are used to encapsulate the shared processes and shared information in a blockchain network. These instructions assume you are already familiar with the concept of a [peer](https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/peers/peers.html) and provides guidance for the various decisions you will have to make about a peer you will deploy and join to a production Fabric network channel. If you need to quickly stand up a network for education or testing purposes, check out the [Fabric test network](../test_network.html).
+
+## Generate peer identities and Membership Service Providers (MSPs)
+
+Before proceeding with this topic, you should have reviewed the process for a [Deploying a Certificate Authority (CA)](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/ca-deploy-topology.html) for your organization in order to generate the identities and MSPs for the admins and peers in your organization. To learn how to use a CA to create these identities, check out [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/{BRANCH}/deployguide/use_CA.html)
+
+**Note that the “cryptogen” tool should never be used to generate any identities in a production scenario**.
+
+### Folder management
+
+While it is possible to bootstrap a peer using a number of folder structures for your MSPs and certificates, we do recommend a particular [folder structure](https://hyperledger-fabric-ca.readthedocs.io/en/{BRANCH}/deployguide/use_CA.html#decide-on-the-structure-of-your-folders-and-certificates) for the sake of consistency and repeatability. These instructions will presume that you have used that folder structure.
```

Comment:
I might have given you the wrong link here. It should point to here instead:
 https://hyperledger-fabric-ca.readthedocs.io/en/{BRANCH}/deployguide/use_CA.html#folder-structure-for-your-org-and-node-admin-identities